ui/app/components/clients/date-range.hbs

```diff
@@ -4,35 +4,44 @@
 }}
 
 <div ...attributes>
-  <Hds::Text::Display @tag="p" class="has-bottom-margin-xs">
-    Client counting period
-  </Hds::Text::Display>
-  <Hds::Text::Body @tag="p" @color="faint" @size="300">
-    The dashboard displays client count activity during the specified date range below. Click edit to update the date range.
-  </Hds::Text::Body>
+  <div class="is-flex-column align-items-end">
+    <Hds::Text::Display @tag="p" @size="100" class="has-bottom-margin-xs">
+      Select billing period
+    </Hds::Text::Display>
+    <div class="is-flex">
+      {{#if (and @startTime @endTime)}}
+        <div class="flex align-items-center">
+          <p class="is-size-5 has-text-weight-semibold" data-test-date-range="start">{{this.formattedDate @startTime}}</p>
+          <p class="has-left-margin-xs"> — </p>
+          <p class="is-size-5 has-text-weight-semibold has-left-margin-xs" data-test-date-range="end">{{this.formattedDate
+              @endTime
+            }}
+          </p>
+        </div>
+      {{/if}}
+      {{#if this.version.isEnterprise}}
+        <Hds::Dropdown class="has-left-margin-xs" @matchToggleWidth={{true}} as |D|>
+          <D.ToggleButton @text="Current Billing Start" @color="secondary" />
```

Comment:
Since historical periods are here as well, this should probably be updated to "Billing start date" (and sentence cased - which is what our current app pattern is for buttons/dropdowns)